php-checks/src/main/java/org/sonar/php/checks/utils/FunctionUsageCheck.java

```diff
@@ -41,6 +43,25 @@ public void visitFunctionCall(FunctionCallTree tree) {
     super.visitFunctionCall(tree);
   }
 
+  protected static boolean argumentIsNullOrEmptyString(FunctionCallTree tree, int argumentIndex) {
+    if (tree.arguments().size() > argumentIndex) {
+      ExpressionTree valueArgument = tree.arguments().get(argumentIndex);
+      return isNullOrEmptyString(valueArgument);
+    }
+    return true;
+  }
+
+  private static boolean isNullOrEmptyString(ExpressionTree tree) {
```

Comment:
I think I'll move this is `CheckUtils` when I'll refactor `CookieSensitiveDataCheck` to use `FunctionUsageCheck` in  [SONARPHP-814](https://jira.sonarsource.com/browse/SONARPHP-814) (to avoid duplication)